tableview.go

```diff
@@ -1027,7 +1093,8 @@ func (l *TableView) OnBeforeDraw(fn func(int, int, int, int)) {
 // * rowCount - the number of visible rows
 func (l *TableView) VisibleArea() (firstCol, firstRow, colCount, rowCount int) {
 	firstRow = l.topRow
-	maxDy := l.height - 3
+	maxDy := l.height - l.titleRows - 1
```

Comment:
As I remember, `3` because of `1` for scrollbar. Now, if I understand the MR right, `maxDy` default value is `2`. Why?